micropub.php

```diff
@@ -49,389 +51,419 @@
 
 if ( ! class_exists( 'Micropub' ) ) :
```

Comment:
looks like this change indents the entire file due to this outer condition. we definitely don't want that. :P if wordpress standards say to, i'm ok with disobeying that. can you configure PHPCS to skip this indent?

Reply:
According to WordPress coding standards, it shouldn't be skipped. It appears in a specific example in Best Practices. 

https://developer.wordpress.org/plugins/the-basics/best-practices/

Reply:
understood, but i'm going to choose to disobey their standards in this case.